cmd/registry/controller/local-tasks.go

```diff
@@ -66,6 +66,18 @@ func (task *ExecCommandTask) Run(ctx context.Context) error {
 	if strings.HasPrefix(task.Action.Command, "registry") {
 		fullCmd := strings.Fields(task.Action.Command)
 
+		// force the exec-ed registry instance to use the same server configuration
+		config, err := connection.ActiveConfig()
+		if err != nil {
+			return err
+		}
+		if config.Insecure {
+			fullCmd = append(fullCmd, "--registry.insecure")
+		}
+		if config.Address != "" {
+			fullCmd = append(fullCmd, "--registry.address")
+			fullCmd = append(fullCmd, config.Address)
+		}
```

Comment:
If we want to refactor this, I could imagine instead calling something like `config.Flags()` or `connection.ActiveConfigFlags()` to get an array of strings that could be appended to `fullCmd`.

Reply:
@shrutiparabgoogle I'm going to merge this on @theganyo's approval, but PTAL at this code and if it seems worth doing, we can move this code into the config package with a small refactor.